be1-go/channel/reaction/mod_test.go

```diff
@@ -544,11 +545,10 @@ func Test_DeleteReaction_Out_of_Order(t *testing.T) {
 
 	// Wait before storing a new message to be able to have a unique
 	// timestamp for each message
-	time.Sleep(time.Millisecond)
+	time.Sleep(9 * time.Millisecond)
```

Comment:
I have to ask, why 9 and not 10 ? 😅

Reply:
Oh that might have been something i forgot to refactor. I will do a (hopefully) final commit that should give something cleaner.